sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportClient.cs

```diff
@@ -59,6 +60,7 @@ internal abstract class TransportClient : IAsyncDisposable
         /// <param name="identifier"></param>
         /// <param name="sessionId"></param>
         /// <param name="isSessionReceiver"></param>
+        /// <param name="transactionGroup"></param>
```

Comment:
not a public class, but this still makes me twitch.   If we're not planning to fill them out, I'd suggest removing them entirely.